src/main/java/mate/academy/service/ManufacturerServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package mate.academy.service;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.ManufacturerDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Manufacturer;
+
+@Service
+public class ManufacturerServiceImpl implements ManufacturerService {
+
+    @Inject
+    private ManufacturerDao manufacturerDao;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        return manufacturerDao.create(manufacturer);
+    }
+
+    @Override
+    public Optional<Manufacturer> getById(Long manufactureId) {
+        return manufacturerDao.getById(manufactureId)get();
```

Comment:
```suggestion
    public Manufacturer getById(Long manufactureId) {
        return manufacturerDao.getById(manufactureId).get();
```
this code doesn't work

Reply:
Sorry. It was Optional.of(manufacturerDao.getById(manufactured).get());
Don't know how it happend, maybe it`s because i was in a hurry.
Changed to your suggestion.